pkg/types/gcp/validation/platform.go

```diff
@@ -67,6 +67,16 @@ var (
 	}()
 )
 
+const (
+	// maxUserLabelLimit is the maximum userLabels that can be configured as defined in openshift/api.
+	// https://github.com/openshift/api/commit/ae73a19d05c35068af16c9aeff375d0b7c936a8a#diff-07b264a49084976b670fb699badaca1795027d6ea732a99226a5388104f6174fR592-R602
+	maxUserLabelLimit = 32
+
+	// MaxUserTagLimit is the maximum userTags that can be configured as defined in openshift/api.
+	// https://github.com/openshift/api/commit/ae73a19d05c35068af16c9aeff375d0b7c936a8a#diff-07b264a49084976b670fb699badaca1795027d6ea732a99226a5388104f6174fR604-R613
+	MaxUserTagLimit = 50
```

Comment:
These constants were first defined in https://github.com/openshift/installer/pull/7126, MaxUserTagLimit is exported here, hence include in this PR as well, will resolve the conflicts, when 7126 is merged.